types/src/config.rs

```diff
@@ -292,6 +292,13 @@ pub struct StakeholderProportion {
     pub vesting: Option<VestingSchedule>,
 }
 
+#[derive(Debug, Eq, PartialEq, Clone)]
+#[near(serializers = [borsh, json])]
+pub enum VestingScheme {
```

Comment:
I guess it would be good to add a detailed description for the enum and for every variant as well.